src/jobs/jobs.controller.ts

```diff
@@ -749,11 +737,11 @@ export class JobsController {
     const ability = this.caslAbilityFactory.createForUser(
       request.user as JWTUser,
     );
-    const canCreate =
+    const canGet =
       ability.can(AuthOp.JobReadAny, JobClass) ||
       ability.can(AuthOp.JobReadAccess, currentJobInstance);
-    if (!canCreate) {
-      throw new ForbiddenException("Unauthorized to update this dataset");
+    if (!canGet) {
+      throw new ForbiddenException("Unauthorized to get this job.");
```

Comment:
```
      const canView =
        ability.can(Action.DatasetReadAny, DatasetClass) ||
        ability.can(Action.DatasetReadOneOwner, datasetInstance) ||
        ability.can(Action.DatasetReadOneAccess, datasetInstance) ||
        ability.can(Action.DatasetReadOnePublic, datasetInstance);

      if (!canView) {
        throw new ForbiddenException("Unauthorized access");
      }
```
I suggest we either follow the variable namings for can ability from the `datasets.controller.ts`, or just to change all namings with canRead, canCreate. Having a vairious naming for the samething makes confusion sometimes.

Reply:
Covered in commit 4aedbc27d5667476a6d87cef1e06194d2be1b6df